influxdb_client/client/write_api.py

```diff
@@ -183,21 +183,24 @@ def __init__(self, influxdb_client, write_options: WriteOptions = WriteOptions()
     def write(self, bucket: str, org: str = None,
               record: Union[
                   str, List['str'], Point, List['Point'], dict, List['dict'], bytes, List['bytes'], Observable] = None,
-              write_precision: WritePrecision = DEFAULT_WRITE_PRECISION) -> None:
+              write_precision: WritePrecision = DEFAULT_WRITE_PRECISION, data_frame_measurement_name: str = None,
+              data_frame_tag_columns: List['str'] = None) -> None:
```

Comment:
What do you think about  use a `**kwargs` instead named args?

https://www.geeksforgeeks.org/args-kwargs-python/